src/test/rpc/Subscribe_test.cpp

```diff
@@ -307,6 +308,85 @@ class Subscribe_test : public beast::unit_test::suite
         BEAST_EXPECT(jv[jss::status] == "success");
     }
 
+    void
+    testTransactions_APIv2()
```

Comment:
Would it be good to rename the original `testTransactions` to `testTransactions_APIv1` so there's no confusion?